### ISSUE TITLE
Can't run v1.14.2 on Windows 11

### ISSUE THREAD

**Issue by IanMichaelHarper** [cryptoadvance/specter-desktop#2032]
I'm in the middle of something so I can't add details right now, but I will try add them later. I just thought I'd flag the issue in case you can reproduce yourselves before I get a chance to add more info.

Also, Avast Antivirus flags this version very aggresively to the point where you have to wait ages for it to do its thing. It even sent it off to its Threat Labs for extra analysis, whatever that is. I'm not sure why this version gets flagged so heavily and v1.14.1 doesn't 🤷🏻‍♂️

**Comment by k9ert** [comment]
So this is only in regards of Avast Antivirus? It's not unusual that new binaries are treated a bit more aggressive than known ones.
Can you add details here?

**Comment by IanMichaelHarper** [comment]
No the application doesn't run on Windows 11 at all.

I'll add details as soon as I can

**Comment by k9ert** [comment]
Unfortunately, i can't reproduce as i don't have a windows 11 available. In the newest version `v1.14.5` i've migrated the python stack to 3.10 and that might make a difference. Can you test that version?

**Comment by IanMichaelHarper** [comment]
Yup, that worked. Antivirus still flags it but I imagine that's hard to get around for most crypto applications
